program/convert-days-to-hours/convert-days-to-hours.rs

```diff
@@ -0,0 +1,19 @@
+use std::io;
+
+fn main() {
+    println!("Enter the number of days: ");
+
+    let mut days = String::new();
+    io::stdin().read_line(&mut days).expect("Failed to read input");
+
+    let days: u32 = match days.trim().parse() {
+        Ok(num) => num,
+        Err(_) => {
+            println!("Please enter a valid number of days");
+            return;
```

Comment:
Code is throwing error irrespective of valid input .

Reply:
Works fine
![Screenshot from 2023-01-11 21-37-11](https://user-images.githubusercontent.com/54644599/211856513-9b999c52-0a01-445f-838f-826bbf78a2dd.png)